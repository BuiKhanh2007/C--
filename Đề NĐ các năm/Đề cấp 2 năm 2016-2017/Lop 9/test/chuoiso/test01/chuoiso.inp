3755 30
6
9306
9506
5621
4163
4439
2152
5763
1427
6380
9740
9161
5415
9907
5428
92
3767
7175
9645
7203
3494
9027
8272
4821
4995
9764
431
8264
1447
8489
