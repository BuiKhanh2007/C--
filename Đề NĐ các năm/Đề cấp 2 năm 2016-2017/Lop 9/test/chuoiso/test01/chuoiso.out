0
1
7
5
1
5
9
1
8
1
1
2
1
7
8
0
8
1
2
1
1
1
1
2
1
7
1
2
5
1
